.gitignore

```diff
@@ -6,4 +6,5 @@ dist
 coverage
 docs
 .vscode
-package-lock.json
\ No newline at end of file
+package-lock.json
+.prettierrc
```

Comment:
Is this on purpose?

Reply:
I locally had a config I diddnt want to push yet.